index.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="description" content="HABASM | HABASM365, Capstone" />
+    <meta name="keywords" content="Web design and Development, Capstone,Front-End,Front-End Developer,HABASM" />
+    <meta name="author" content="Capstone" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="stylesheet" href="css/desktop.css" />
+    <title>Green energy suitability</title>
+  </head>
+  <body>
+    <nav class="smallMenu">
+      <ul class="darkMenu">
+        <li>
+          <a href="#"
+            ><i class="fa fa-facebook social" aria-hidden="true"></i
+          ></a>
+        </li>
+        <li>
+          <a href="#"
+            ><i class="fa fa-twitter social" aria-hidden="true"></i
+          ></a>
+        </li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Profile</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </nav>
+    <nav class="bigMenu">
+      <ul>
+        <li>
+          <a class="logo" href="#"
+            ><img src="img/greenenergy.png" alt="Logo image"
+          /></a>
```

Comment:
- [ ] when the mobile popup menu comes up, the user should not be able to scroll through the background until a link is clicked or menu is closed. please fix that.

![Screenshot (615)_LI](https://user-images.githubusercontent.com/65298048/203500737-5f021d68-3a8d-42d3-9051-c94ae512c29a.jpg)